Changes

```diff
@@ -14,31 +14,12 @@ of the usual "- ".
 NEXT_RELEASE:
 -------------
 
-- "noautolink" tag for ocaml{c,opt}
-  (Gabriel Scherer)
-
-0.10.1 (26 Dec 2016):
----------------------
-
-- #124, #136: do not explicitly pass -shared when building shared libraries;
-  let the compiler decide what to build.
-  (whitequark)
-
-- #137, #138: fix the man/ installation for non-opam users
-  (Adam Sampson and Gabriel Scherer)
-
-- fix `--version` output: 0.10.0 displays "0.9.3"...
-  (Gabriel Scherer)
-
-0.10.0 (25 Dec 2016):
----------------------
-
-OCamlbuild 0.10.0 introduces a minor change to the way `.cmxs` files
-are produced when no `.mldylib` file is absent: it will now use the
-exact same semantics as `.cmxa` and `.mllib` file -- in particular, it
+OCamlbuild 0.11.0 introduces a change to the way `.cmxs` files are
+produced when no `.mldylib` file is absent: it will now use the exact
+same semantics as `.cmxa` and `.mllib` file -- in particular, it
 should not be necessary anymore to have identical
 `foo.{mllib,mldylib}` files, only `foo.mllib` should suffice. See the
-detailed Changelog for details.
+detailed Changelog below for details.
```

Comment:
Nit: the file is called "Changes" (was it not?), so this should probably be "detailed change log".